File: JavaScript/Exercicio28.js

function contarVogais(frase) {
    let vogais = 0;
    for(let i=0; i<frase.length; i++) {
        let letra = frase[i];
        if(letra == "a" || letra == "e" || letra == "i" || letra == "o" || letra == "u") {
            vogais += 1;
        }
    }
    return vogais;
}

console.log(contarVogais("Bora Programar"));

File: CSharp/Exercicio28.cs

public int ContarVogais(string frase)
{
    int vogais = 0;
    for(int i =0; i<frase.Length; i++)
    {
        char letra = frase[i];
        if(letra == 'a'|| letra == 'e' || letra == 'i' || letra == 'o' || letra == 'u')
        {
            vogais ++;
        }
    }
    return vogais;
}

Console.WriteLine(ContarVogais("Bora Programar"))